suites/reef/nvmeof/tier-2_nvmeof_4nodes_gateway_ha_tests.yaml

```diff
@@ -1,5 +1,5 @@
 # Basic Ceph-NvmeoF sanity Test suite
-# cluster configuration file: conf/reef/nvmeof/ceph_nvmeof_sanity.yaml
+# cluster configuration file: conf/reef/nvmeof/ceph_nvmeof_ha_cluster_4nodes.yaml
```

Comment:
Please add the same config changes into squid folder aswell.

Reply:
Done. added all changes from reef to squid test suite as well.